README.md

```diff
@@ -14,202 +14,81 @@
 # limitations under the License.
  -->
 
-# MaxText
+# **MaxText**
```

Comment:
Nit: why do we need to make all these bold when they are already behind a heading level?